plugins/modules/postgresql_ext.py

```diff
@@ -72,9 +72,9 @@
     description:
       - Extension version to add or update to. Has effect with I(state=present) only.
       - If not specified, the latest extension version will be created.
-      - It can't downgrade an extension version.
-        When version downgrade is needed, remove the extension and create new one with appropriate version.
-      - Set I(version=latest) to update the extension to the latest available version.
+      - Version downgrade is only supported if extension provides a downgrade path.
+        Otherwise extension must be removed and lower version of extension must be made available.
```

Comment:
```suggestion
      - Downgrading is only supported if the extension provides a downgrade path otherwise the extension must be removed and a lower version of the extension must be made available.
```